### ISSUE TITLE
Alien Archive browser loading error

### ISSUE THREAD

**Issue by TheGreatRedDuke** [foundryvtt-starfinder/foundryvtt-starfinder#971]
**Describe the bug**
Clicking the Alien Archive button at the bottom of the Compendiums panel results in a error and never loads (stops at 3/4 in the loading bar).

**To Reproduce**
Click the Alien Archive button.

**Expected behavior**
Loads the Alien Archive browser.

**Screenshots**
![Screenshot 2023-02-14 112534](https://user-images.githubusercontent.com/78890368/218797360-62ca3ebc-6262-47ec-a305-697ad1162f47.jpg)

**Technical details (please complete the following information):**
 - OS: Windows 11
 - Browser: Chrome
 - Foundry Version: 10.291
 - SFRPG Version: 0.22.1

**Additional context**
```
foundry.js:747 TypeError: An error occurred while rendering AlienArchiveBrowserSFRPG 96. Cannot read properties of undefined (reading 'cr')
[Detected 1 package: system:sfrpg]
    at AlienArchiveBrowserSFRPG.loadItems (alien-archive-browser.js:50:62)
    at async AlienArchiveBrowserSFRPG.getData (document-browser.js:204:26)
    at async AlienArchiveBrowserSFRPG._render (foundry.js:5108:18)
onError	@	foundry.js:747
🎁call_wrapped	@	libWrapper-wrapper.js:507
🎁Hooks.onError#lib-wrapper	@	listeners.js:138
🎁Hooks.onError#0	@	libWrapper-wrapper.js:187
(anonymous)	@	foundry.js:5069
Promise.catch (async)		
render	@	foundry.js:5067
(anonymous)	@	browsers.js:25
dispatch	@	jquery.min.js:2
y.handle	@	jquery.min.js:2
```

**Comment by LebombJames** [comment]
Not sure why everyone is finding this bug all of a sudden when there hasn't been a release in over a month 😆

Run these to macros in order:
```js
await game.settings.set("sfrpg","alienArchiveBrowser",`{}`)
```
```js
await game.settings.set("sfrpg","alienArchiveBrowser",`{"sfrpg.alien-archives": {"load":true, "name": "Alien Archives"}}`)
```
This will remove any custom packs from your compendium browser, and will need to be run every game launch. Working on a fix.

**Comment by LebombJames** [comment]
Was closed by #972
